GliaWidgets/ViewController/EngagementViewController.swift

```diff
@@ -32,19 +32,19 @@ class EngagementViewController: ViewController, AlertPresenter {
         viewModel.event(.viewDidDisappear)
     }
 
-    func showBackButton(with style: HeaderButtonStyle, in header: Header) {
-        let back = HeaderButton(with: style, tap: { self.viewModel.event(.backTapped) })
-        back.contentHorizontalAlignment = .left
-        header.setLeftItem(back, animated: false)
+    func showBackButton(in header: Header) {
+        header.backButton.tap = { self.viewModel.event(.backTapped) }
+        header.setLeftItem(animated: false)
     }
 
-    func showCloseButton(with style: HeaderButtonStyle, in header: Header) {
-        let back = HeaderButton(with: style, tap: { self.viewModel.event(.closeTapped) })
-        back.contentHorizontalAlignment = .right
-        header.setRightItem(back, animated: false)
+    func showCloseButton(in header: Header) {
+        header.closeButton.tap = { self.viewModel.event(.closeTapped) }
+        header.showCloseButton()
     }
```

Comment:
I think we can remove both of these methods entirely. Just move the `header.***button.tap` line to `bind()`, just as you did with the others, and call the `header.showCloseButton()` and `header.showBackButton()` (or however you will rename it) in `bind(...)` methods of `CallViewController` and `ChatViewController`.